flask_jwt_extended/view_decorators.py

```diff
@@ -220,6 +222,24 @@ def _decode_jwt_from_query_string():
     return decode_token(encoded_token)
 
 
+def _decode_jwt_from_json(request_type):
+    if request.content_type != 'application/json':
+        raise NoAuthorizationError('Invalid content-type. Must be application/json.')
+
+    if request_type == 'access':
+        token_key = config.json_key
+    else:
+        token_key = config.refresh_json_key
+
+    try:
+        encoded_token = request.json.get(token_key, None)
+        assert encoded_token
```

Comment:
I like the control flow here, but assertions can actually be disabled on the python project when running python with the -O flag, so I think this should section should be reworked without the use of assert.